File: Loon-0.5-Alpha/Loon-Neo/src/loon/Platform.java
package loon;

public interface Platform {

	public static enum Orientation {
		Portrait, Landscape;
	}
	
	public abstract void close();

	public abstract int getContainerWidth();

	public abstract int getContainerHeight();

	public abstract Orientation getOrientation() ;

	public abstract LGame getGame() ;
}

File: C#/Loon2MonoGame/LoonMonoGame-Lib/loon/Platform.cs
using Microsoft.Xna.Framework.Content;
using Microsoft.Xna.Framework.Graphics;

namespace loon
{
	public interface Platform
	{

		void Close();

		int GetContainerWidth();

		int GetContainerHeight();

		Platform_Orientation GetOrientation();

		LGame GetGame();

	}

	public enum Platform_Orientation
	{
		Portrait,
		Landscape
	}
}
